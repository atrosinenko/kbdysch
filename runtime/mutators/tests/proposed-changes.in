[ENV:KBDYSCH_MUTATOR_NUM_SPLICES=0]
[BEGIN:DUMPING OUTPUTS]
[END:CALL DEINIT]
[MUTATE]
%P14ABCDEFGH1234567890
%P24ABCDEFGH1234567890
%P83ABCDEFGH1234567890
