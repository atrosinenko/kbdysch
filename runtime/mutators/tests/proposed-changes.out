INJECTOR: Mutator requested 3 mutations.
INJECTOR: OUTPUT #0: BEGIN
INJECTOR: %P14ABCDAFGH1234567890
INJECTOR: %P24ABCDEFGH1234567890
INJECTOR: %P83ABCDEFGH1234567890
INJECTOR: OUTPUT #0: END
INJECTOR: OUTPUT #1: BEGIN
INJECTOR: %P14ABCDEFGH1234567890
INJECTOR: %P24ABCDABGH1234567890
INJECTOR: %P83ABCDEFGH1234567890
INJECTOR: OUTPUT #1: END
INJECTOR: OUTPUT #2: BEGIN
INJECTOR: %P14ABCDEFGH1234567890
INJECTOR: %P24ABCDEFGH1234567890
INJECTOR: %P83ABCABCDEFGH4567890
INJECTOR: OUTPUT #2: END
